.github/workflows/build-artifact.yaml

```diff
@@ -21,7 +21,7 @@ on:
       arch:
         required: false
         type: string
-        default: '["grayskull", "wormhole_b0"]'
+        default: '["wormhole_b0"]'
```

Comment:
Do we want to default to `wormhole_b0` only?

Reply:
Ah shit didn't meant to put that there - will fix